scripts/03001-walz.js

```diff
@@ -0,0 +1,7 @@
+const HNG_ID = '03001';
+const full_name = 'Izaak Chukwuma';
+const language = 'JavaScript';
+
+console.log(
+  `Hello World, this is ${full_name} with HNGi7 ID hng-${HNG_ID} using ${language} for stage 2 task `
```

Comment:
Please use upper case for the HNG in your HNG-ID

Reply:
Change hng-id to HNG-ID I also added my email address

Reply:
The change he meant is like so "HNG-12345" within the string